t/t0402-block-command-on-gvfs.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+test_description='block commands in GVFS repo'
+
+. ./test-lib.sh
+
+not_with_gvfs () {
+    command=$1 &&
+    shift &&
+    test_expect_success "test $command $*" "
+        test_config alias.g4rbled $command &&
+        test_config core.gvfs true &&
+        test_must_fail git $command $* &&
+        test_must_fail git g4rbled $* &&
+        test_unconfig core.gvfs &&
+        test_must_fail git -c core.gvfs=true $command $* &&
+        test_must_fail git -c core.gvfs=true g4rbled $*
+    "
+}
+
+not_with_gvfs fsck
+not_with_gvfs gc
+not_with_gvfs gc --auto
+not_with_gvfs prune
+not_with_gvfs repack
+not_with_gvfs submodule status
+not_with_gvfs update-index --index-version 2
+not_with_gvfs update-index --skip-worktree
+not_with_gvfs update-index --no-skip-worktree
+not_with_gvfs update-index --split-index
+not_with_gvfs worktree list
+
+test_expect_success 'test gc --auto succeeds when disabled via config' '
```

Comment:
I like this test! Let's also get VFS functional tests before we merge. We will also want to retest the CI builds after #90 merges.

Reply:
The VFS functional tests are available in this PR:
https://github.com/Microsoft/VFSForGit/pull/597